amethyst_network/src/systems/network_client.rs

```diff
@@ -0,0 +1,50 @@
+extern crate ron;
+
+use specs::{Entities, Entity, Join, System, WriteStorage};
+use std::net::UdpSocket;
+use std::net::IpAddr;
+use std::str;
+
+use amethyst_core::transform::*;
+
+use ::components::netsync::*;
+
+pub struct NetClientSystem {
+    pub socket:UdpSocket,
```

Comment:
Please use whitespace after the field name, it looks very confusing without

Reply:
fixed everywhere I saw that happening